src/main/resources/application.yml

```diff
@@ -13,6 +13,8 @@ application:
       some_val: "Hello World"
     query:
       another_val: 123
+  finalResponse:
+      httpStatusCode: 200
```

Comment:
Update Readme.md as well, including with how to not set a default httpStatusCode